### PR TITLE
PAPP-24016 Resolve Django vulnerability on py36

### PR DESCRIPTION
**Pull request by paulnawat**
* Resolve Django vulnerability on py36
* Django 4.0.2 only works on Py38 and up. As a result, we need to change the version to 3.2.12 to remove the Whitesource vulnerabilities in py36

### CODE REVIEW

**Comment by paulnawat** [comment]
Green py3.9 pipeline: https://cd.splunkdev.com/phantom-apps/app-tests/-/pipelines/4523127

File: office365.json [hunk 1]
```diff
@@ -4723,7 +4723,7 @@
         "wheel": [
             {
                 "module": "Django",
-                "input_file": "wheels/py3/Django-3.2.5-py3-none-any.whl"
```
Comment (paulnawat) [inline office365.json:4726]: We get rid of 3.2.5 vulnerable version entirely here.

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
 beautifulsoup4==4.9.1
-Django==4.0.2
+Django==3.2.12
```
Comment (paulnawat) [inline requirements.txt:2]: https://pypi.org/project/Django/3.2.12/

File: office365.json (hunk 1 above)
Comment (paulnawat) [reply office365.json:4726]: It's previously kept b/c 4.0.2 only works on py3.8+ 

See https://pypi.org/project/Django/4.0.2/

**Review by gary-phantom — APPROVED** [review]

File: office365.json [hunk 3]
```diff
@@ -4805,6 +4805,10 @@
                 "module": "python_magic",
                 "input_file": "wheels/shared/python_magic-0.4.18-py2.py3-none-any.whl"
             },
+            {
+                "module": "pytz",
+                "input_file": "wheels/shared/pytz-2021.3-py2.py3-none-any.whl"
+            },
```
Comment (paulnawat) [inline office365.json:4811]: this package is added by running `pre-commit run --all-files`

**Pull request merged**